org/istio.yaml

```diff
@@ -647,7 +645,6 @@ orgs:
               WG - Docs Maintainers/Infra:
                 description: Maintainers of the doc build & site infrastructure
                 maintainers:
-                - geeknoid
```

Comment:
@sdake  can you be a maintainer here?

Reply:
@mandarjog happy to help, although I am not sure what the implications are after reading @howardjohn's message.

As an organization, we should limit github admins to the bare minimum. This isn't about trust (thanks for the vote of confidence though!), but concerns about attacks on someone's 2FA. I recently linked an article in #security where an attack on someone's 2FA was used to steal hundreds of thousands of dollars in bitcoin from the coinbase.com platform.

Such an attack could be used with github.com as well. I am not saying someone would do this, but could do this, resulting in the deletion of repositories. Such an action might not be deadly for the project, but it would be highly damaging, and I wouldn't want a 2FA attack confused with intent on my part.

Personally i think (multiple) redundant robots should be used for github admin. I am not aware of the need to use github admin privileges, but github admin should be highly limited for the reason above.

Cheers
-steve

Reply:
reference: https://medium.com/coinmonks/the-most-expensive-lesson-of-my-life-details-of-sim-port-hack-35de11517124

Reply:
And a recommendation for anyone with github admin ACLs, please please disable any phone SMS recovery method. There are other ways to enable a backup method.

I don't know who these individuals are, but @howardjohn should and perhaps you can communicate this message among the github admins.

I have disabled my SMS 2FA after re-reading that article. Good practice for anyone in the Istio organization, but especially for those with github superpowers.

Cheers,
-steve